src/esp/gfx/Renderer.cpp

```diff
@@ -79,7 +82,26 @@ struct Renderer::Impl {
                 esp::gfx::Renderer::Impl::RendererShaderType::
                     DepthTextureVisualizer);
 
+#ifdef CORRADE_TARGET_APPLE
+        Mn::Image2D image = tgt.getDepthTexture().image(
+            0, {Mn::GL::PixelFormat::DepthComponent, Mn::GL::PixelType::Float});
+        Mn::ImageView2D imgView{image.storage(), Mn::PixelFormat::R32F,
+                                image.size(), image.data()};
```

Comment:
```suggestion
        Mn::GL::BufferImage2D image = tgt.getDepthTexture().image(
            0, {Mn::GL::PixelFormat::DepthComponent, Mn::GL::PixelType::Float}, GL::BufferUsage::StaticRead);
        Mn::Vector2i size = image.size(); // because release() clears the size/storage, it has to be fetched first
        Mn::PixelStorage storage = image.storage();
        // This takes `image` (which is depth) and "reinterprets" it as R32F
        Mn::GL::BufferImage2D imgView{ // shouldn't be named imgView anymore I guess?
            storage, Mn::PixelFormat::R32F, size, image.release());
```

... I hope it can work directly like this, worst-case with minor corrections.